osu.Game.Rulesets.Osu/Objects/Drawables/Pieces/SpinnerDisc.cs

```diff
@@ -73,6 +73,9 @@ public bool Complete
             }
         }
 
+        public bool Valid => spinner.StartTime <= Time.Current && spinner.EndTime > Time.Current;
+        public bool Trackable { get; set; } = true;
```

Comment:
I'm not sure this is the proper way to go. It solves the problem, but is yet another flag to take into account inside of the spinner, and doesn't really make sense semantically (what's the meaning of a spinner being *trackable* or not?). It'd be cleaner if the disc could stay unmodified at all.

Not sure what the right answer I'd be looking here is. Manual `TriggerEvent`s, maybe? Some kind of proxying input manager that would only send events to the spinner (which I'm not even sure is viable)? I have no idea myself, so this comment isn't a change request *per se*.